### DISCUSSION TITLE
Hello, other version of Windows 10:)

### DISCUSSION THREAD

**Discussion by Reejaybp** [farag2/Sophia-Script-for-Windows#359] [Q&A] [answered]
For the version 21H2 (19044.1766) what script i need it for?

**Comment by Reejaybp** [comment]
![imagen_2022-07-04_083041807](https://user-images.githubusercontent.com/108702604/177228237-386d60da-c148-49fa-8239-ed4535ba745c.png)

**Comment by farag2** [comment, marked as answer]
For Windows 10 of course:)
